src/pages/iou/steps/MoneyRequestAmountForm.js

```diff
@@ -217,14 +231,20 @@ function MoneyRequestAmountForm({amount, currency, isEditing, forwardedRef, onCu
                         longPressHandlerStateChanged={updateLongPressHandlerState}
                     />
                 ) : null}
-                <Button
-                    success
-                    style={[styles.w100, styles.mt5]}
-                    onPress={submitAndNavigateToNextPage}
-                    pressOnEnter
-                    isDisabled={!currentAmount.length || parseFloat(currentAmount) < 0.01}
-                    text={buttonText}
-                />
+                <View style={[styles.w100, styles.mt5]}>
```

Comment:
What does this look like on mobile where we show the full keypad too?

Reply:
I feel like we should consider moving that error message to be below the green button for consistency. I think it's probably okay if the UI jumps slightly because the alternatives aren't really too great. We'd have to give a an area a fixed height to prevent jumping, which kind of feels like we'd be optimizing for the less common case. 

Also, we should make the error text use our text.Supporting color, but keep the dot red.

Reply:
But then that makes an awkward gap between the green button and the keypad. So I think for this one, we should move the error message to be below the green submit button.